sidedish/src/component/ItemDetail/ItemDetail.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import styled from 'styled-components';
+import usePortal from 'hooks/usePortal';
+import { createPortal } from 'react-dom/cjs/react-dom.development';
```

Comment:
혹시 development에서 import하신 이유가 있으실까요? 개발용에는 무관하지만 배포용에서는 번들사이즈가 커지는 이슈가 발생할 수 있을 것 같습니다.

```suggestion
import { createPortal } from 'react-dom';
```

Reply:
미션을 시작할 때 배포까지는 염두에 두지 않았지만,
말씀해주신 부분은 완전히 모르고 있던 부분이네요..!!
감사합니다!!

Reply:
auto import에서 실수로 한것 같습니다. 어떤 이슈로 이어질 수 있는지도 알려주셔서 감사합니다!